scripts/check-imports.js

```diff
@@ -41,7 +41,7 @@ let exitCode = 0;
 main();
 
 function main() {
-  globSync('packages/mdc-*/**/*.js')
+  globSync('packages/**/*.js')
```

Comment:
Was there a reason that this was changed?

Reply:
Yes. That glob wasn't including packages/material-components-web/index.js, and all of these checks for imports should apply there as well.